cypress/support/commands.js

```diff
@@ -360,6 +177,111 @@ Cypress.Commands.add('findSubmissionAsEditor', (username, password, familyName,
 	cy.contains('View ' + familyName).click({force: true});
 });
 
+Cypress.Commands.add('createSubmissionWithApi', (data, csrfToken) => {
```

Comment:
Really good to have some API-based data seeding patterns to build on. Not to mention API tests in the bargain.